routes/contactUs/helpers.js

```diff
@@ -0,0 +1,27 @@
+import emailJs from "emailjs";
+
+export default {
+    contactUsmail: async (req, res) => {
+        try {
+            const server = await emailJs.server.connect({
+                host:    "smtp.gmail.com",
+                password:"agroeasy1$##",
+                ssl:     true,
+                user:    "agroeasy2018@gmail.com",
+            });
+            const { email, name, message, subject } = req.body;
+            const subjectToUpperCase = subject.toUpperCase();
+            
+            const userMessage = {
+                from: email,
+                subject: subjectToUpperCase,
+                text:    `${name} (${email}) says: ${message}`,
+                to: "agroeasy2018@gmail.com",
+            };
+        
+            await server.send(userMessage, (err, message) => res.json({ message, success: true }));
```

Comment:
given that you are using `await` here. This should be more like
```
const message = await server.send(userMessage);
res.json({ message, success: true })
```

Reply:
```here, server.send() takes two parameters.
1. message from the user
2. a callback function that takes error and message.```

const message = await server.send(userMessage);
`message`  is undefined here